Controller/RegistrationFOSUser1Controller.php

```diff
@@ -59,16 +58,25 @@ public function registerAction()
                 $route = $this->get('session')->get('sonata_basket_delivery_redirect');
 
                 if (null !== $route) {
+                    // NEXT_MAJOR: remove the if block
+                    $deprecationMessage = <<<'EOT'
```

Comment:
Héhé, I know… but I couldn't get vim to color this file normally without it :/ I'll try harder.